.travis.yml

```diff
@@ -7,6 +7,16 @@ env:
     TORTOISE_TEST_MODULES=tests.testmodels
 matrix:
   include:
+  - name: Python 3.7 / Windows / Partial
+    os: windows
+    language: shell
+    before_install:
+      - choco install python --version 3.7.4
+      - python -m pip install --upgrade pip
+      - choco install make
+    env: PATH=/c/Python37:/c/Python37/Scripts:$PATH
+    script: make testall
```

Comment:
This will try and test all the db variants? I suspect we may not have them all available right now.
It's fine to start with just `make test` to only test SQLite.

Reply:
Yeah. I'll try with the smaller one.